src/tests/test_magic_cmd.py

```diff
@@ -416,9 +417,29 @@ def test_test_error(ip, cell, error_type, error_message):
     assert str(out.error_in_exec) == error_message
 
 
-def test_snippet(ip_snippets):
+@pytest.mark.parametrize(
+    "precmd, result",
+    [
+        (
+            "",
+            Table(
+                ["Stored snippets"],
+                [["high_price"], ["high_price_a"], ["high_price_b"]],
+            ),
+        ),
+        (
+            "%sqlcmd snippets -d high_price_a",
```

Comment:
Deleting a snippet might affect some other tests. To make sure we are not, I think you can create a new snippet and see if it's in the table. then, clean up these snippets. I would test the following cases: no snippets, 1, 2, and 3 snippets.

Reply:
@yafimvo 
To ensure it doesn't affect other tests, I created a new `@pytest.fixture` for this test function for creating and deleting snippets.